toolbackup/pom.xml

```diff
@@ -170,12 +170,14 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
+            <version>1.11.83</version>
         </dependency>
 
         <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-core -->
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-core</artifactId>
+            <version>1.11.83</version>
```

Comment:
Odd, these two should be in our bom via the AWS bom 
https://github.com/dockstore/dockstore/blob/develop/bom-internal/pom.xml#L337-L342

Reply:
Looking at the [package names](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/migration-whats-different.html), our bom has the AWS version 2 SDK, but toolbackup uses version 1